### ISSUE TITLE
redirect after file upload to voucher link

### ISSUE THREAD

**Issue by yasermahmood** [filesender/filesender#1332]
Hi Team,

there is an issue when we create a link to use voucher to transfer file via web interface as follows:
![image](https://user-images.githubusercontent.com/6165820/197738849-2922f339-6da5-4725-8fce-1e5d603d258a.png)

and when user use voucher link to upload file **once successfully done it again redirect it to voucher URL**, if option is checked "valid for one upload only " it generate an error message like this since URL is valid for one upload only

![image](https://user-images.githubusercontent.com/6165820/197739487-c72fbd7e-a26f-4bc9-bc05-13960ff4b1bc.png)

**Comment by WebSpider** [comment]
We're seeing this in 2.36 as well. 
The issue is cause specifically because the redirect after upload is not just to `home`, but it contains the `vid` parameter, which refers to a now expired voucher.

**Comment by liedekef** [comment]
The current simple fix for this:

go in www/upload_page.js and change the close-function ar around line 1163, from:

```
        var close = function() {
            window.filesender.notification.clear();
            filesender.ui.goToPage(
                filesender.ui.transfer.guest_token ? 'home' : 'transfers',
                reditectargs,
                filesender.ui.transfer.guest_token ? null : 'transfer_' + filesender.ui.transfer.id
            );
        };
```
to
```
        var close = function() {
            window.filesender.notification.clear();
            filesender.ui.goToPage(
                filesender.ui.transfer.guest_token ? 'home' : 'transfers',
                filesender.ui.transfer.guest_token ? null : reditectargs,
                filesender.ui.transfer.guest_token ? null : 'transfer_' + filesender.ui.transfer.id
            );
        };
```

**Comment by monkeyiq** [comment]
I have merged the solution by @liedekef in https://github.com/filesender/filesender/pull/1343 and that is part of release 2.38.

**Comment by WebSpider** [comment]
This is indeed fixed in our 2.38 test environment.
@yasermahmood can you see if 2.38 fixes this for you as well?

**Comment by monkeyiq** [comment]
Thank you for the independent feedback. Always nice to know it is working in a real installation :)

**Comment by madsi1m** [comment]
> This is indeed fixed in our 2.38 test environment. @yasermahmood can you see if 2.38 fixes this for you as well?

It does for us, cheers
